src/components/CommentManager.jsx

```diff
@@ -1,29 +1,84 @@
 import { useState, useEffect } from "react";
 import CommentList from "./CommentList";
-
-import { getCommentsByArticleId } from "../../api";
+import { getCommentsByArticleId, postCommentByArticleId } from "../../api";
+import { useContext } from "react";
+import { UserContext } from "./UserProvider";
 
 export default function CommentManager({ article_id }) {
   const [comments, setComments] = useState();
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState(null);
+  const [newComment, setNewComment] = useState({
+    author: "",
+    body: "",
+    votes: 0,
+    created_at: "2020-01-26T18:23:00.000Z",
+  });
+  const { loggedInUser, setLoggedInUser } = useContext(UserContext);
+  const [isSuccess, setIsSuccess] = useState(false);
 
   useEffect(() => {
     getCommentsByArticleId({ article_id })
       .then((data) => {
         setComments(data.data);
+
         setIsLoading(false);
       })
       .catch((err) => {
         setError({ err });
       });
   }, []);
 
+  function handleChange(e, property) {
+    setNewComment({
+      ...newComment,
+      [property]: e.target.value,
+      ["author"]: "jessjelly", //loggedInUser.username,
+    });
+  }
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    setComments((c) => {
```

Comment:
let's make this `c` a bit more descriptive such as `currentComments` would be a better and more descriptive variable name